src/components/Checkout/OrderSummary.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState, useEffect } from 'react';
+import {
+	Heading,
+	VStack,
+	SimpleGrid,
+	GridItem,
+	Text,
+	Button,
+	Divider,
+} from '@chakra-ui/react';
+import { colors } from '../../util/constants';
+
+import { useStripe, useElements } from '@stripe/react-stripe-js';
```

Comment:
There should be a better to way to organize imports for readability.

Reply:
can you elaborate?

Reply:
For example, imports for components from React libraries can come first, then our custom components, and finally custom constants/config files.

Reply:
ok i see